features/docs/cli/publish_banner.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Publish banner
+
+  A banner is displayed on stderr at the end of a run to inform that reports can
+  be published on reports.cucumber.io[reports.cucumber.io](https://reports.cucumber.io).
+
+  Scenario: Banner is displayed when running Cucumber
+    Given a file named "features/hello.feature" with:
+    """
+    Feature: Hello
+    """
+    When I run `cucumber`
+    Then the output should contain:
+    """
+    ┌──────────────────────────────────────────────────────────────────────────┐
+    │ Share your Cucumber Report with your team at https://reports.cucumber.io │
+    │                                                                          │
+    │ Command line option:    --publish                                        │
+    │ Environment variable:   CUCUMBER_PLUGIN_PUBLISH_ENABLED=true             │
```

Comment:
Change to `CUCUMBER_PUBLISH_ENABLED` when https://github.com/cucumber/cucumber-jvm/pull/2089 is merged

Reply:
done